jobsets/cardano.nix

```diff
@@ -29,7 +28,7 @@ let
   };
 in pkgs.lib.fix (jobsets: {
   inherit (jobs) iohk-ops iohk-ops-integration-test nixops;
-  inherit (iohkpkgs) checks IFDPins cachecache;

```

Comment:
the removal of `IFDPins` may make caching worse
it might be better to write a reusable function that can generate the pins from the niv `sources` attr